Content.Shared/Alert/AlertType.cs

```diff
@@ -52,7 +52,8 @@ public enum AlertType : byte
         SuitPower,
         BorgHealth,
         BorgCrit,
-        BorgDead
+        BorgDead,
+        Offer
```

Comment:
```suggestion
        Offer,
```
Always use trailing commas, so the diffs look better in the future